org.eclipse.lsp4e.debug/src/org/eclipse/lsp4e/debug/launcher/JsonParserWithStringSubstitution.java

```diff
@@ -0,0 +1,87 @@
+package org.eclipse.lsp4e.debug.launcher;
```

Comment:
Please add a copyright header (same as other filesbut with  updated year and your copyright owner (most likely your employer)